ios/Swift/README.md

```diff
@@ -1,17 +1,66 @@
-# Image detection with iOS device photos
+# Cloud Vision Swift Sample
 
 This app demonstrates how to use the [Cloud Vision API](https://cloud.google.com/vision/) to run label and face detection on an image.
 
-## Local Setup
+## Prerequisites
+- An API key for the Cloud Vision API (See
+  [the docs][getting-started] to learn more)
+- An iOS device running iOS 9.0 or above
+- [Xcode 7][xcode]
+- Install [CocoaPods][cocoapods] if you don't have it already by running the command `sudo gem install cocoapods`.
```

Comment:
I assume the use of 'gem' means they need to have ruby installed too? So add a preq of something like 'have ruby and gem installed'. Though for most that should already be the case.